docs/usage-notes.md

```diff
@@ -54,6 +55,30 @@ There are 2 methods to achieve this:
   - Note: the GUI menus may differ slightly on different Linux distributions.
 - From the command line: Use `chmod +x CATcher-x.y.z.AppImage`
 
+
+# Customising the available sessions 
+
+If your sessions are not present in the default dropdown list on CATcher's startup page, you can load custom sessions by clicking on the **file icon** beside the session dropdown. 
+
+Following which, submit a json file with the name `profiles.json`, where the format is specified below.
+
+```json
+{
+    "profiles": [
+        {
+            "profileName": "CATcher", 
+            "encodedText": "CATcher-org/public_data"
+        }
+    ]
+}
+```
+
+The json supplied should only consist of **one key-pair value**, where the key is `"profiles"` and the value is an array of `Profiles` supplied, where each `Profile` is an object containing the `profileName` and `encodedText` fields. 
+
+`profileName` refers to the profile name displayed in the session select page. `encodedText` refers to the repository which stores the required settings for your custom session. 
```

Comment:
Encoded text technically represents `organisation/repository` instead of just repository. I'm not sure if we need to be specific about this but if no need then this sounds good! 👍

Reply:
No harm to be as specific as possible HAHA